nifi-nar-bundles/nifi-cassandra-bundle/nifi-cassandra-processors/src/main/java/org/apache/nifi/processors/cassandra/QueryCassandra.java

```diff
@@ -492,6 +492,8 @@ protected static String getJsonElement(Object value) {
             SimpleDateFormat dateFormat = new SimpleDateFormat("yyyy-MM-dd HH:mm:ssZ");
             dateFormat.setTimeZone(TimeZone.getTimeZone("UTC"));
             return "\"" + dateFormat.format((Date) value) + "\"";
+        } else if (value instanceof String) {
+            return "\"" + ((String) value).replaceAll("\"", "\\\\\"") + "\"";
```

Comment:
Should we use StringEscapeUtils.escapeJson() here?

Reply:
Of course, thanks! I updated the PR.